syntheseus/reaction_prediction/inference/gln.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, *args, dataset_name: str = "schneider50k", **kwargs) -> None:
 
             self.model = RetroGLN(self.model_dir, chkpt_path)
 
+    @property
+    def name(self) -> str:
+        return "GLN"
```

Comment:
The base class does `self.__class__.__name__.removesuffix("Model")`, but `removesuffix` does not exist in Python 3.7...